<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>play</class>
 <widget class="QWidget" name="play">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>輕鬆一下</string>
  </property>
  <property name="windowIcon">
   <iconset resource="play_data.qrc">
    <normaloff>:/play_data/icon.ico</normaloff>:/play_data/icon.ico</iconset>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>48000</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/car_map.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/game_name.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="car">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/car.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_start_game">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/play_data/start_btn.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>160</width>
     <height>49</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_distance">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體 Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>奔跑距離:  0</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_clock">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="block_01">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item3.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_02">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item3.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_03">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_04">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_05">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_06">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_15">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item3.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_19">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_18">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_16">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item3.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_21">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_20">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_11">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_17">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item3.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_12">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_13">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_14">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="block_22">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="play_data.qrc">:/play_data/item2.png</pixmap>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>block_03</zorder>
  <zorder>block_01</zorder>
  <zorder>block_02</zorder>
  <zorder>block_04</zorder>
  <zorder>block_05</zorder>
  <zorder>block_06</zorder>
  <zorder>block_15</zorder>
  <zorder>block_19</zorder>
  <zorder>block_18</zorder>
  <zorder>block_16</zorder>
  <zorder>block_21</zorder>
  <zorder>block_20</zorder>
  <zorder>block_10</zorder>
  <zorder>block_11</zorder>
  <zorder>block_17</zorder>
  <zorder>block_12</zorder>
  <zorder>block_13</zorder>
  <zorder>block_14</zorder>
  <zorder>block_22</zorder>
  <zorder>car</zorder>
  <zorder>label_distance</zorder>
  <zorder>lcd_clock</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_title</zorder>
  <zorder>button_start_game</zorder>
 </widget>
 <resources>
  <include location="play_data.qrc"/>
 </resources>
 <connections/>
</ui>
